requirements.txt

```
@@ -1,4 +1,4 @@
-singer-python==5.9.0
+singer-python==5.12.1
```

To the project maintainers - would you consider accepting the following change instead 🙏 ?

```suggestion
singer-python~=5.12.1
```

I think this would reduce the potential for dependency hell